library/network/src/modules/firewalld_wrapper.rb

```diff
@@ -32,7 +32,7 @@ module Yast
   class FirewalldWrapperClass < Module
     include Logger
 
-    VALID_PROTOCOLS = ["udp", "tcp"].freeze
+    VALID_PROTOCOLS = ["udp", "tcp", "sctp", "dccp"].freeze
```

Comment:
well I'd really be careful here. See ```/etc/protocols```

Reply:
This four are what firewalld support, this is just a wrapper we really need to support more?

Reply:
https://github.com/yast/yast-http-server/blob/f4b0fbd8b20e4f58b3d199473d3d6fc28dbf0b91/src/modules/YaPI/HTTPD.pm#L1276